app/internal/page/guidance.go

```diff
@@ -3,17 +3,20 @@ package page
 import (
 	"net/http"
 
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/actor"
+
 	"github.com/ministryofjustice/opg-go-common/template"
 	"github.com/ministryofjustice/opg-modernising-lpa/internal/validation"
 )
 
 type guidanceData struct {
-	App    AppData
-	Errors validation.List
-	Lpa    *Lpa
+	App                 AppData
+	Errors              validation.List
+	Lpa                 *Lpa
+	CertificateProvider *actor.CertificateProvider
 }
 
-func Guidance(tmpl template.Template, lpaStore LpaStore) Handler {
+func Guidance(tmpl template.Template, lpaStore LpaStore, certificateProviderStore CertificateProviderStore) Handler {
```

Comment:
I think this should be moved down and duplicated in `page/donor` and `page/certificateprovider`, then you don't have passing `nil`s in for one of the args.